wolfssh/internal.h

```diff
@@ -404,6 +415,7 @@ struct WOLFSSH_CTX {
     byte* privateKey;                 /* Owned by CTX */
     word32 privateKeySz;
     byte useEcc;                      /* Depends on the private key */
+    byte useSaber;                    /* Depends on the private key */
```

Comment:
Please wrap with `#ifndef WOLFSSH_NO_SABER_LEVEL1_SHA256`. Also these should be bit-fields. `byte useSaber:1;` and `byte useEcc:1;`. You'll also need all the code with `!ssh->handshake->useSaber)` wrapped in the macro. I don't want code size to grow if saber is not used.